docker-compose.yml

```
@@ -42,9 +42,10 @@ services:
       # Override ENSADMIN_PUBLIC_URL to point to docker compose ensindexer
       # Note: it must be URL accessible from web browser (i.e. it cannot be a hostname in docker network)
       ENSADMIN_PUBLIC_URL: http://localhost:4173
-      # Override NEXT_PUBLIC_PREFERRED_ENSNODE_URL to docker compose ensindexer
-      # Note: it must be URL accessible from web browser (i.e. it cannot be a hostname in docker network)
-      NEXT_PUBLIC_PREFERRED_ENSNODE_URL: http://localhost:42069
+      # Override NEXT_PUBLIC_DEFAULT_ENSNODE_URLS to docker compose ensindexer
+      # Note: it must be a comma-separated list of URLs that are accessible from web browser
+      # (i.e. it cannot be a hostname in docker network)
+      NEXT_PUBLIC_DEFAULT_ENSNODE_URLS: http://localhost:42069,https://api.alpha.ensnode.io
```

```suggestion
      NEXT_PUBLIC_DEFAULT_ENSNODE_URLS: http://localhost:42069
```

Suggest we just include this one instance in the list.

Can you please verify this comma separated list supports the case of only 1 in the list (and therefore no comma)?